makefiles/tools/dfu.inc.mk

```diff
@@ -2,7 +2,14 @@ DFU ?= dfu-util
 FLASHER ?= $(DFU)
 HEXFILE = $(BINFILE)
 
-FFLAGS ?= -D $(HEXFILE) --reset $(DFU_ARGS)
+ifdef ROM_OFFSET
+  FLASH_ADDR = $(shell printf "0x%x" $$(($(ROM_START_ADDR) + $(ROM_OFFSET))))
+else
+  FLASH_ADDR = $(ROM_START_ADDR)
+endif
 
-RESET ?= $(DFU)
```

Comment:
This change removed the `RESET` for the `pyboard`.

Reply:
I know and I think this wasn't tested in #11080 because reset with dfu cannot work. In #11080, it was certainly a left-over from a copy/paste of `edbg.inc.mk`.

Reply:
Please then put it in its own commit and would need testing it too.

Reply:
Actually this was never needed since rebooting a board using DFU (which is possible) requires to reset the board with the buttons correctly pushed to put it in DFU mode. Thus, the command doesn't make sense in this scenario.